packages/generator/ni_measurement_plugin_sdk_generator/client/templates/measurement_plugin_client.py.mako

```diff
@@ -27,7 +27,7 @@ from pathlib import Path
 <%
     typing_imports = ["Any", "Generator", "List", "Optional"]
     if output_metadata:
-        typing_imports += ["NamedTuple"]
+        typing_imports += ["NamedTuple", "Sequence"]
     if "from pathlib import Path" in built_in_import_modules:
         typing_imports += ["Iterable"]
```

Comment:
This is wrong. Using paths does not necessary require Iterable. Using array input parameters requires Iterable.

Reply:
@Jotheeswaran-Nandagopal @Avinash2Suresh This is too complicated. I'm going to replace it with `import typing` and `import pathlib`.

Reply:
Done.